### ISSUE TITLE
accessibility: set role="main" on template

### ISSUE THREAD

**Issue by rahulbot** [c4fcm/DataBasic#89]
> 
All pages should have a "main" element or a container with role="main" attribute.  This helps  screen reader users determine the main focus of the page and quickly navigate to the most pertinent content.
